frontend/modules/sequences/sequences.selectors.ts

```diff
@@ -189,13 +184,22 @@ export const selectSelectedStateId = createSelector(
 	selectSelectedFrame, (frame) =>  (frame || {}).state
 );
 
+export const selectSelectedFrameViewpoint = createSelector(
+	selectSelectedFrame,  (frame) =>  (frame || {}).viewpoint
+);
+
 export const selectLastSelectedStateId = createSelector(
 	selectLastSelectedFrame, (frame) =>  (frame || {}).state
 );
 
-export const selectIsLoadingFrame = createSelector(
-	selectSelectedStateId, selectStateDefinitions,
-	(stateId, stateDefinitions) => !(stateDefinitions || {}).hasOwnProperty(stateId)
+export const selectIsViewpointFrame = createSelector(
+	selectSelectedFrame, (frame) => (frame || {}).hasOwnProperty('viewpoint')
```

Comment:
Is it better to just piggy back on L187? i.e.

```suggestion
	selectSelectedFrameViewpoint, (viewpoint) => !!viewpoint
```